deployments/bridges/rialto-millau/docker-compose.yml

```
@@ -85,9 +87,12 @@ services:
       LETSENCRYPT_HOST: grafana.millau.brucke.link,grafana.rialto.brucke.link
       LETSENCRYPT_EMAIL: admin@parity.io
     volumes:
-       - ./bridges/rialto-millau/dashboard/grafana/provisioning/:/etc/grafana/provisioning/
+       - ./bridges/rialto-millau/dashboard/grafana/relay-millau-to-rialto-headers-dashboard.json:/etc/grafana/provisioning/dashboards/relay-millau-to-rialto-headers-dashboard.json
+       - ./bridges/rialto-millau/dashboard/grafana/relay-millau-to-rialto-messages-dashboard.json:/etc/grafana/provisioning/dashboards/relay-millau-to-rialto-messages-dashboard.json
+       - ./bridges/rialto-millau/dashboard/grafana/relay-rialto-to-millau-headers-dashboard.json:/etc/grafana/provisioning/dashboards/relay-rialto-to-millau-headers-dashboard.json
+       - ./bridges/rialto-millau/dashboard/grafana/relay-rialto-to-millau-messages-dashboard.json:/etc/grafana/provisioning/dashboards/relay-rialto-to-millau-messages-dashboard.json
 
   prometheus-metrics:
     volumes:
-      - ./bridges/rialto-millau/dashboard/prometheus/:/etc/prometheus/
+      - ./bridges/rialto-millau/dashboard/prometheus/prometheus.yml:/etc/prometheus/prometheus-rialto-millau.yml
```

```suggestion
      - ./bridges/rialto-millau/dashboard/prometheus/prometheus.yml:/etc/prometheus/rialto-millau.yml
```
Is `prometheus-` prefix adding any value here?

---

EDIT: I see we glob-import all `prometheus-*.yml` files, so maybe better to use `job-rialto-millau.yml` to avoid confusion with the main `prometheus.yml` configuration file?

---

It isn't job, it's a set of targets. There's single job now

---

Then a `target-` prefix maybe? My point is to avoid confusion about using `prometheus.yml`-level options in that file (cause as you've already mentioned to Hernando they are not valid in this context).